XenModel/XenAPI-Extensions/Network.cs

```diff
@@ -125,13 +125,20 @@ public bool AutoPlug
             {
                 // Note that this is not equivalent to BoolKey, because here
                 // absence of the key gives AutoPlug=true, not false.
-                return Get(other_config, "automatic") != "false";
+                // CA-218956 - Expose HIMN when showing hidden objects

```

Comment:
I would prefer to leave the Autoplug property unchanged (i.e. read/write the automatic flag regardless of the network type) and prevent this being set/used if the network is HIMN in XenCenter, for example by hiding the checkbox from the Network Properties dialog (although that wouldn't really be necessary if the Properties dialog would never be accessible for HIMN)